sysbuild/Kconfig.suit

```diff
@@ -31,14 +31,75 @@ config SUIT_ENVELOPE_ROOT_TEMPLATE_FILENAME
 config SUIT_ENVELOPE_SIGN_SCRIPT
 	string "Location of SUIT sign script"
 	depends on SUIT_ENVELOPE_SIGN
+	default "modules/lib/suit-generator/ncs/sign_script.py"
```

Comment:
`${ZEPHYR_SUIT_GENERATOR_MODULE_DIR}`?

Reply:
No - currently we use the relative path - I will switch to the absolute path in the PR responsible for build system changes for signing, together with other required changes.

Note in this PR I've only moved the line according to Jamie's suggestion, I didn't add anything